sccm/apps/deploy-use/import-export-applications.md

```diff
@@ -0,0 +1,46 @@
+---
+
+# Mandatory fields.
+title: Import/Export Applications
+titleSuffix: Configuration Manager
+description: Import and export applications.
+author: chriskibble
+ms.author: NO VALUE SET # Microsoft employees only
+ms.date: 5/6/2019
+ms.topic: conceptual
+ms.technology: configmgr-app
+---
+# Create applications in Configuration Manager
+
+*Applies to: System Center Configuration Manager (Current Branch)*
+
+Configuration Manager allows for the import and export of applications within the console.  This allows you to copy applications in to other environments, such as copying from a test environment into a production environment.
+
+## <a name="bkmk_export"></a> Export an Application
+
+1. Within the Configuration Manager console, right click on the application to export and select **Export**.
+1. On the **General** screen, enter a path to a new ZIP file to export into.  Optionally define if you'd like to export *dependencies, supersedence relationships, conditions, and virtual environments* and also if you'd like to export *content for the selected applications and dependencies*.  Enter any administrator comments, if desired, and click **Next**.
```

Comment:
Same note as above in the next few lines regarding use of "click". I generally use "select" unless it gets repetitive or just sounds better to use "choose"

Reply:
Fixed in #1585